src/dmd/json.d

```diff
@@ -83,11 +91,10 @@ public:
         buf.writeByte('\"');
     }
 
-    void stringPart(const(char)* s)
+    extern(D) void stringPart(const(char)[] s)
```

Comment:
Wouldn't all non-visit members of this visitor be private? Ergo you can just add `extern(D):` at the top of this body rather than adding it for each individual function.

Think about it for future refactor.

Reply:
True that. At the moment I'm putting `extern(D)` right and left (got more slice-ification coming), but I also had weird bugs (compiler SEGV) being triggered in the past some I decided to go for verbose and simple. But when enough is converted, we surely can do that.